src/openrct2.c

```diff
@@ -59,6 +59,7 @@ utf8 gCustomPassword[MAX_PATH] = { 0 };
 bool gOpenRCT2Headless = false;
 
 bool gOpenRCT2ShowChangelog;
+bool gOpenRCT2SilentBreakpad;
```

Comment:
Isn't it better to define this in crash.cpp if you have defined the extern in crash.h?